datadog_checks_base/datadog_checks/base/config.py

```diff
@@ -13,7 +13,7 @@ def is_affirmative(value):
 
     # use object's implementation of `__bool__`, it's faster than cast.
     # None -> False, 0 -> False, 1 -> True, etc.
-    return not not value
+    return bool(value)
```

Comment:
This is on purpose, see the comment just above.

Reply:
i can change it back if you want, from the test i did it's seams to be faster the bool() than the 'not not'

Reply:
I am changing it right away ;-)